platform/security/docs/src/main/resources/IntContents.adoc

```diff
@@ -1426,3 +1426,58 @@ The `AnonymousInterceptor` is installed by default with DDF Security Application
 . Click save.
 
 Once these configurations have been added, the AnonymousInterceptor is ready for use. Both secure and non-secure requests will be accepted by all secure DDF service endpoints.
+
+== Security IdP
```

Comment:
If this is the first section we are mentioning IdP I would recommend spelling out what it stands for. Not sure if we cover that in some acronym section or elsewhere.